tests/utils.py

```diff
@@ -494,10 +517,14 @@ async def __call__(  # pylint: disable=invalid-overridden-method, useless-super-
 }
 
 
-def get_st_init_args(recipe_list: List[Any]) -> Dict[str, Any]:
+def get_st_init_args(recipe_list: List[Any], extra_config: Optional[Dict[str, Any]]=None) -> Dict[str, Any]:
+    if extra_config is None:
```

Comment:
Where is this used?

Reply:
To decrease the number of redundant lines while doing supertokens init. The default params for core endpoint, app name, etc remain same across all the tests. I import and use this function to get those args..